src/services/Auth/logInHandler.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+//LOGIN
+export const logInHandler = ({ email, pass }, setUser, navigator) => {
```

Comment:
Since we don't usually use states in services. If you avoid using states in services that would be good.